### DISCUSSION TITLE
How to add color animation effect on text ?

### DISCUSSION THREAD

**Discussion by ProPhycient5** [tailwindlabs/tailwindcss#11295] [Help] [answered]
I am attaching the link below, where there is a showcase of a button having `background-color` as smoothly transitioned (animated gradient bg-color).  Credit @Pouya Saadeghi (creator of daisyUI). I want to add a similar animated feature on the `color` of any text. 
https://play.tailwindcss.com/kVPuB4NVdf?file=css
Please help🙏🏾

**Comment by wongjn** [comment]
It does not seem like you have attached a link.

**Reply by ProPhycient5** [reply]
Sorry @wongjn , I missed it. Now, I have updated.

**Reply by wongjn** [reply, marked as answer]
1. Apply the conic gradient background:
   ```html
   <p class="… bg-[conic-gradient(from_0deg_at_50%_100%,hsl(260deg_40%_40%),hsl(260deg_50%_20%),hsl(0deg_0%_0%),hsl(260deg_50%_20%),hsl(260deg_40%_40%))]">
   ```
2. Limit the background to the length of the text:
   ```html
   <p class="… w-max">
   ```
3. Show the background through the text:
   ```html
   <p class="… bg-clip-text text-transparent">
   ```
4. Make the angle of the conic gradient able to be interpolated for animations:
   ```css
   @property --angle {
     syntax: '<angle>';
     inherits: false;
     initial-value: 0deg;
   }
   ```
   ```html
   <p class="… bg-[conic-gradient(from_var(--angle)_at_50%_100%,hsl(260deg_40%_40%),hsl(260deg_50%_20%),hsl(0deg_0%_0%),hsl(260deg_50%_20%),hsl(260deg_40%_40%))]">
   ```
5. Add animation to to the angle of the conic gradient:
   ```js
   module.exports = {
     theme: {
       extend: {
         animation: {
           fancy: 'fancy 6s infinite linear',
         },
         keyframes: {
           fancy: {
             from: {
               '--angle': '0deg',
             },
             to: {
               '--angle': '360deg',
             },
           },
         },
       },
     },
   };
   ```
   ```html
   <p class="… animate-fancy">
   ```

Live example at https://play.tailwindcss.com/WMSCEWSRrT.

**Reply by ProPhycient5** [reply]
Yes @wongjn that's what I wanted. You are a genius !!!!!👏👏👏👏👏🙏🏼 .
How do you gotta learn so many in-depth concepts?
